test_infra/ess/deployment.tf

```diff
@@ -1,7 +1,13 @@
 variable "stack_version" {
   type        = string
   default     = "latest"
-  description = "the stack version to use"
+  description = "the stack deployment version"
+}
+
+variable "stable_snapshot_version" {
```

Comment:
why we need this new var? pinning the images at the a "stable" tag is already supported by setting them inside `pkg/testing/ess/create_deployment_csp_configuration.yaml`, right? 🤔

Reply:
Do you mean we can update the `pkg/testing/ess/create_deployment_csp_configuration.yaml` versions instead of extracting the version to another file?